index.html

```diff
@@ -0,0 +1,1161 @@
+                </div>
+              </div>
+              <div class="card rounded-0 border-0 mt-5" >
+                <div class="position-relative p-0">
+                  <img src="images/content/mario-diaz-balart.jpg" class="card-img-top" alt="donald-trump.jpg">
+                  <p class="position-absolute py-1 px-3 text-uppercase bg-light text-danger" style="bottom: -16px;">u.s.</p>
+                </div>
+                <div class="card-body p-4 border">
+                  <h5 class="card-title">Florida Rep Is First of Two Congressmen to Test Positive for Coronavirus</h5>
+                  <p class="card-text _arial _fnt15 text-dark">""I want everyone to know that I am feeling much better," Rep. Diaz-Balart said Wednesday in a statement.</p>
+                </div>
+              </div>
+              <div class="card rounded-0 border-0 mt-5" >
+                <div class="position-relative p-0">
+                  <img src="images/content/donald-trump.jpg" class="card-img-top" alt="donald-trump.jpg">
+                  <p class="position-absolute py-1 px-3 text-uppercase bg-light text-danger" style="bottom: -16px;">u.s.</p>
+                </div>
+                <div class="card-body p-4 border">
+                  <h5 class="card-title">Trump Signs Emergency Bill to Manufacture Medical Supplies for Coronavirus</h5>
+                  <p class="card-text _arial _fnt15 text-dark">"It is critical that all health and medical resources needed to respond to the spread of COVID-19 are properly distributed to the Nation's healthcare system and others that need them most at this time," read Trump's executive order.</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div> 
+      </div>
+    </div>
+    <!-- IN THE MAGAZINE -->
+    <div class="container-fluid px-0 mx-0 mt-3">
+      <div class="border-bottom _fnt18 text-center pb-2 mx-3">
+        <img src="images/favicon-28.png" alt=".." class="mr-2">
+        IN THE MAGAZINE
+      </div>
+      <div class="mt-3 px-0 mx-0">
+        <div class="container-fluid py-3 px-0">
+          <div class="row p-0 m-0 mw-100 justify-content-between">
+            <div class="col-md-3">
+              <div class="card rounded-0 border-0 py-3">
+                <div class="position-relative p-0">
+                  <img src="images/content/bernie.jpg" class="card-img-top" alt="donald-trump.jpg">
+                </div>
+                <div class="card-body p-0 py-2 text-left">
+                  <div class="my-2">
+                    <span class="bg-danger text-white px-2 py-1 ">FEATURE</span><span class="ml-1 px-2 py-1 text-danger">HEALTH</span>
+                  </div>
+                  <h5 class="card-title">Is Bernie Sanders Right About Medicare For All?</h5>
+                </div>
+              </div>
+            </div>
```

Comment:
- [ ] The first element of this section has a different layout (it's larger than the rest and has no title). Please check the images below and make the necessary changes:
![Screenshot from 2020-03-23 16-17-38](https://user-images.githubusercontent.com/51926937/77368473-e7e34100-6d21-11ea-8fad-265cc4f21535.png)

Original:
![Screenshot from 2020-03-23 16-17-44](https://user-images.githubusercontent.com/51926937/77368480-ea459b00-6d21-11ea-981b-bae04eb9fb07.png)

- [ ] Check the dimensions of the articles' images in this section: they should have a larger height in all breakpoints except xs and sm.

Reply:
Added a different Card element for that markup part